src/css/grid.css

```diff
@@ -0,0 +1,233 @@
+.span-1-of-1 {
```

Comment:
WHy 2 grid style files. REview the whole folder

Reply:
The grid.css was generated from grid.scss.
These all are generated files. I have removed them and created a single css file main.css.